pkgs/development/libraries/kde-frameworks/kinit/default.nix

```diff
@@ -14,7 +14,6 @@ mkDerivation {
     kconfig kcrash ki18n kio kservice kwindowsystem
   ];
   patches = [
-    ./0001-kinit-libpath.patch
```

Comment:
Is it okay to drop the patch?

I'd want the patch's author (@ttuegel) to review. Things relied on `NIX_PROFILES` for a while, not sure if it still does, or can. And not sure how this can or would break with `nix-env` installed or `nix-shell` ran KDE apps.

Reply:
The commit that removed that part of the code doesn't seem to have added any replacement, that's why I removed the patch completely:
https://github.com/KDE/kinit/commit/be91fc21edcaaba1ac8fdb590febac7b32226174

Reply:
Some more background reading, linked from the commit you linked:

 - https://phabricator.kde.org/T12140

I see this must mean it's just not possible to do what this did. Discussion resolved then.